crt_portal/cts_forms/templates/forms/base.html

```diff
@@ -99,8 +100,10 @@
     </div>
     {% endblock %}
   </body>
+  {% compress js %}
   <script src="{% static 'js/url_params_polyfill.js' %}"></script>
   <script src="{% static 'js/uswds.min.js' %}"></script>
   <script src="{% static 'js/focus_alert.js' %}"></script>
   {% block page_js %}{% endblock %}
```

Comment:
Note that most of our per-template JS is in a `page_js` block, so this captures a whole lot of other linked javascript files and inline javascript.